tmt/steps/execute/internal.py

```diff
@@ -25,6 +30,21 @@ class ExecuteInternal(tmt.steps.execute.ExecutePlugin):
         tmt.steps.Method(name='beakerlib.tmt', doc=__doc__, order=80),
         ]
 
+    REBOOT_VARIABLE = "REBOOT_COUNT"
```

Comment:
I'd move the constants to the top and just replace the token when creating the wrappers.